scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -567,42 +566,42 @@ integrated_sce
 
 There are couple pieces of information here of interest:
 
-- The `corrected` reduced dimension represents the batch-corrected PCA.
-- The `reconstructed` assay represents the batch-corrected normalized expression values, which `fastMNN` "back-calculated" from the batch-corrected PCA.
-Generally speaking, these are not stand-alone values that you should use for other applications like differential gene expression.
-If the `subset.row` argument was provided (as we did here), only genes present in `subset.row` are included in these reconstructed expression values, but this can be overridden so that all genes have reconstructed expression with the argument `correct.all = TRUE`.
+- The `corrected` reduced dimension represents the batch-corrected PCA that `fastMNN` calculated. 
+- The `reconstructed` assay represents the batch-corrected normalized expression values, which `fastMNN` "back-calculated" from the batch-corrected PCA (`corrected`).
+Generally speaking, these expression values are not stand-alone values that you should use for other applications like differential gene expression.
+If the `subset.row` argument was provided (as it was here), only genes present in `subset.row` are included in these reconstructed expression values, but this can be overridden so that all genes have reconstructed expression with the argument `correct.all = TRUE`.
 
 We're mostly interested in the PCA that `fastMNN` calculated, so let's save that information (with an informative and unique name!) into our `merged_sce` object:
 
 ```{r fastmnn pcs}
-# Make a new reducedDim named fastMNN_PCA from the corrected reducedDim in integrated_sce
-reducedDim(merged_sce, "fastMNN_PCA") <- reducedDim(integrated_sce, "corrected")
+# Make a new reducedDim named fastmnn_PCA from the corrected reducedDim in integrated_sce
+reducedDim(merged_sce, "fastmnn_PCA") <- reducedDim(integrated_sce, "corrected")
 
 merged_sce
 ```
 
-Finally, we'll calculate UMAPs from these corrected PCs and plot them so we can see how qualitatively successful integration was:
-
+Finally, we'll calculate UMAP from these corrected PCA matrix for visualization.
 ```{r fastmnn umap}
 # Calculate UMAP
 merged_sce <- scater::runUMAP(
   merged_sce, 
   # create UMAP from this PCA matrix:
-  dimred = "fastMNN_PCA", 
+  dimred = "fastmnn_PCA", 
   # name the UMAP:
-  name = "fastMNN_UMAP"
+  name = "fastmnn_UMAP"
 )
 ```
 
 First, let's plot the integrated UMAP highlighting the different batches.
 A well-integrated dataset will show batch mixing, but a poorly-integrated dataset will show more separation among batches, similar to the uncorrected UMAP.
+Note that this is a more qualitative way to assess the success of integration, but there are formal metrics one can use to assess batch mixing.
```

Comment:
Do we want to link to OSCA for some of these? Maybe just a link to http://bioconductor.org/books/3.15/OSCA.multisample/correction-diagnostics.html?

Reply:
Done w/ a `3.16` link.